Extensions/blacklist.js

```diff
@@ -955,6 +957,14 @@ XKit.extensions.blacklist = new Object({
 									return m_word;
 								}
 							}
+
+							if (thisCodeIsBadAndTheFollowingEscapeIsGarbage.includes(m_word)) {
```

Comment:
move to L953 for clarity and less repetition
`if (tmp_word.includes(m_word) || unsanitised.includes(m_word) {`

Reply:
oh